README.md

```diff
@@ -25,6 +25,7 @@ BIDS currently supports the following data modalities with more to come in the f
 - behavioral
 - physiological
```

Comment:
Good question! I went with capital letter for "Microscopy" as it is the name of the imaging modality itself.
I don't know why behavioral and physiological are in lower-case.
I can make the changes to capital letter in this PR but from a maintaining point of view, it may be best to make them in a separate PR to better keep track.

Reply:
I would keep diff to only pertinent to the goal (microscopy BEP) of this PR, keep it consistent with what is there within BIDS already (thus lower case microscopy below), and any other changes (e.g. styling of listing here to become capitalized) should go into a separate PR.

Reply:
Thank you @yarikoptic, I agree to keep consistency with lower-case for now (including microscopy).

Reply:
Sounds very good! Thanks!